ctk/idp/NotTested.md

```diff
@@ -35,6 +35,7 @@
 | 3.4.1 - Element `<AuthnRequest>` | The identity provider MUST have a trusted means to map the index value in the attribute to information associated with the requester. \[SAMLMeta\] provides one possible mechanism. | Hard to test because we don’t know what method the IdP is going to use.
 | 3.4.1.4 - Processing Rules | In such a case, the identifier's physical content MAY be different, but it MUST refer to the same principal. | Figuring out if the identifiers refer to the same principal cannot be done without actually resolving the identifiers to a principal. For now all that is checked is identical identifier values.
 | 3.4.1.4 - Processing Rules | The assertion(s) returned MUST contain a `<saml:Subject>` element that represents the presenter. | Figuring out if the subject "represents the presenter" cannot be done without actually resolving the subject to a principal. For now all that is checked is that each Assertion in the Response contains a Subject.
+| 3.7.3.2 Session Authority Rules | If \[the session authority\] cannot \[successfully terminates the principal's session\], then it MUST respond with a `<LogoutResponse>` message containing a top-level status code indicating the error.| Initially, we were testing this section by logging out a user which was not logged in. After further analysis we were able to determine that it qualifies as failure during logout.
```

Comment:
✏️ "...cannot [successfully terminates..." reads very oddly. I believe that the text in brackets is directly quoted from the SAML spec so shouldn't be altered, but we might be able to change our verbiage so it reads something like...
`If there is a failure testing that [the session authority] [successfully terminates the...]`

Reply:
It's the other way the spec says 
`If it cannot do so, then it MUST respond with a <LogoutResponse> message containing a top-level status code indicating the error. Thus, the top-level status indicates the state of the logout operation only with respect to the session authority itself.`